### PR TITLE
docs: Deploy App Workloads with a PCG Tutorial - DOC-1004

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR creates the "Deploy App Workloads with a PCG" tutorial.

## Review Changes

💻 [Deploy App Workloads with a PCG](https://deploy-preview-2353--docs-spectrocloud.netlify.app/clusters/pcg/deploy-app-pcg)

🎫 [DOC-1004](https://spectrocloud.atlassian.net/browse/DOC-1004)

[DOC-1004]: https://spectrocloud.atlassian.net/browse/DOC-1004?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 1]
```diff
@@ -0,0 +1,641 @@
...
+target environment. However, a System Private Gateway is recommended when direct network connectivity is available.
+Visit the [System Private Gateway](./architecture.md#system-private-gateway) section to learn more.
+
+A PCG comprises a cluster of nodes and can be deployed by two methods. These include using the
+[Palette CLI](./deploy-pcg/) for VMware vSphere, MAAS, or OpenStack environments, or deploying it
+[manually onto an existing Kubernetes cluster](./deploy-pcg-k8s.md).
+
+In this tutorial, you will deploy a VMware PCG using Palette CLI. Next, you will learn how to deploy a VMware cluster
+with a sample Kubernetes application called
+[Hello Universe](https://github.com/spectrocloud/hello-universe#hello-universe), utilizing either Palette or Terraform.
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:33]: Just clearing up with the dashboard as both really utilise Palette.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 2]
```diff
@@ -0,0 +1,641 @@
...
+authentication with Palette.
+
+```text hideClipboard
+Welcome to Spectro Cloud Palette
+```
+
+The video below demonstrates Palette's authentication process. Ensure you utilize values appropriate for your
+environment.
+
+<Video title="palette-login-video" src="/videos/palette-login.mp4"></Video>
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:111]: This video displays `cardenas88karl-gmail` can we get some blurring on it?

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 3]
```diff
@@ -0,0 +1,641 @@
...
+   | **Node Affinity** | `N`       | No                       |
+
+After answering the prompts of the `pcg install` command, a new PCG configuration file is generated, and its location is
+displayed on the console.
+
+```text hideClipboard
+==== PCG config saved ==== Location: /home/ubuntu/.palette/pcg/pcg-20240306182821/pcg.yaml
+```
+
+Next, Palette CLI will create a local kind cluster that will be used to bootstrap the PCG cluster deployment in your
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:233]: Can we put a link to https://kind.sigs.k8s.io/ to put the word kind into perspective?

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 4]
```diff
@@ -0,0 +1,641 @@
...
+VMware environment. Once installed, the PCG registers itself with Palette and creates a VMware cloud account with the
+same name as the PCG.
+
+The following recording demonstrates the `pcg install` command with the `--config-only` flag. When using this flag, a
+configuration file named **pcg.yaml** is created under the path **.palette/pcg**. You can then utilize this file to
+install a PCG with predefined values using the command `pcg install` with the `--config-file` flag. Refer to the
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:239]: Perhaps a sentence that this provides a reusable PCG installation mind be good?

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 5]
```diff
@@ -0,0 +1,641 @@
...
+[Palette CLI PCG Command](../../palette-cli/commands/pcg.md) page for further information about the command.
+
+<Video title="palette-pcg-config-only" src="/videos/palette-pcg.mp4"></Video>
+
+<br />
+<br />
+
+You can monitor the PCG cluster creation by logging into Palette, switching to the **Tenan Admin** scope, clicking on
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:247]:
```suggestion
You can monitor the PCG cluster creation by logging into Palette, switching to the **Tenant Admin** scope, clicking on
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 6]
```diff
@@ -0,0 +1,641 @@
...
+**Tenant Settings** from the left **Main Menu**, then selecting **Private Cloud Gateways** from the left **Tenant
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:248]: This is getting a bit confusing as you are clicking on different things in one sentence. Can we break this up in separate sentences?

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 7]
```diff
@@ -0,0 +1,641 @@
...
+Next, Palette CLI will create a local kind cluster that will be used to bootstrap the PCG cluster deployment in your
+VMware environment. Once installed, the PCG registers itself with Palette and creates a VMware cloud account with the
+same name as the PCG.
+
+The following recording demonstrates the `pcg install` command with the `--config-only` flag. When using this flag, a
+configuration file named **pcg.yaml** is created under the path **.palette/pcg**. You can then utilize this file to
+install a PCG with predefined values using the command `pcg install` with the `--config-file` flag. Refer to the
+[Palette CLI PCG Command](../../palette-cli/commands/pcg.md) page for further information about the command.
+
+<Video title="palette-pcg-config-only" src="/videos/palette-pcg.mp4"></Video>
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:242]: This video shows `sp-carolina` and `spectrocloud.dev`. Perhaps blur?

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 8]
```diff
@@ -0,0 +1,641 @@
...
+<br />
+
+You can monitor the PCG cluster creation by logging into Palette, switching to the **Tenan Admin** scope, clicking on
+**Tenant Settings** from the left **Main Menu**, then selecting **Private Cloud Gateways** from the left **Tenant
+Settings Menu**. Click on the PCG cluster you just deployed to access its **Overview** page and check the deployment
+progress under the **Events** tab.
+
+![PCG Events page.](/clusters_pcg_deploy-app-pcg_pcg-events.png)
+
+Additionally, you can observe the PCG deployment progress from your terminal. Upon completion, the local kind cluster is
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:254]:
```suggestion
You can also follow along with the PCG deployment progress from your terminal. Upon completion, the local kind cluster is
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 9]
```diff
@@ -0,0 +1,641 @@
...
+**Private Cloud Gateways** from the **Tenant Settings Menu** and check if the PCG you created is listed. Click on your
+PCG and confirm that its cluster status is **Running** and **Healthy**.
+
+![PCG Overview page.](/clusters_pcg_deploy-app-pcg_pcg-health.png)
+
+## Create a Cluster Profile and Deploy a Cluster
+
+Once you have successfully deployed the PCG, proceed to create a cluster profile with the
+[Hello Universe](hhttps://github.com/spectrocloud/hello-universe) application and use it to deploy a VMware cluster.
+This tutorial offers two different workflows: one utilizing the Palette User Interface (UI) and the other employing
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:271]:
```suggestion
You can use the Palette dashboard or 
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 10]
```diff
@@ -0,0 +1,641 @@
...
+
+| Pack Name      | Version | Layer       |
+| -------------- | ------- | ----------- |
+| hello-universe | 1.1.x   | App Service |
+
+Click on the **Confirm & Create** button and then click **Next**.
+
+**Review**
+
+Review the selected cluster layers and configurations. If everything looks correct, click on **Finish Configuration** to
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:342]:
```suggestion
Review the selected cluster layers and configurations. Click on **Finish Configuration** to
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 11]
```diff
@@ -0,0 +1,641 @@
...
+complete the cluster profile creation.
+
+![Cluster profile review page.](/clusters_pcg_deploy-app-pcg_cluster-profile.png)
+
+### Deploy a VMware Cluster
+
+Navigate to the left **Main Menu** and select **Profiles**. Locate and click on the cluster profile you previously
+created. Next, click on the **Deploy** button and confirm by clicking **OK** to initiate the cluster deployment using
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:350]:
```suggestion
created. Next, click on the **Deploy** button and confirm by clicking **OK** to start the cluster deployment using
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 12]
```diff
@@ -0,0 +1,641 @@
...
+Click on the **Confirm & Create** button and then click **Next**.
+
+**Review**
+
+Review the selected cluster layers and configurations. If everything looks correct, click on **Finish Configuration** to
+complete the cluster profile creation.
+
+![Cluster profile review page.](/clusters_pcg_deploy-app-pcg_cluster-profile.png)
+
+### Deploy a VMware Cluster
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:347]: I wonder if this entire deployment flow should be included in this tutorial or if we should defer to another deploy a cluster page https://docs.spectrocloud.com/clusters/data-center/vmware#deploy-a-vmware-cluster

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 13]
```diff
@@ -0,0 +1,641 @@
...
+
+```bash
+terraform plan
+```
+
+The output displays the resources that Terraform will create in an actual implementation, including the cluster profile,
+VMware vSphere cluster, and a new SSH key pair if the keys were not provided.
+
+```text hideClipboard
+# Output condensed for readability
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:516]: I've not seen these lines in any of our tutorials.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 14]
```diff
@@ -0,0 +1,641 @@
...
+Log in to your VMware vSphere server and select the VM representing the PCG node named `gateway-tutorial-cp`. Click on
+the **Three-Dot Actions** button, select **Power**, and **Power Off** to power off the machine. Once the machine is
+powered off, click on the **Three-Dot Actions** button again and select **Delete from Disk** to remove the machine from
+your VMware vSphere environment.
+
+![Delete VMware VM](/clusters_pcg_deploy-app-pcg_vmware-delete.png)
+
+## Wrap-Up
+
+In this tutorial, you learned how to deploy a VMware PCG using Palette CLI and utilize it to deploy a Palette cluster
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:628]:
```suggestion
In this tutorial, you learned how to deploy a VMware PCG using Palette CLI.  Then, you used the PCG to deploy a Palette cluster
```

**Review by addetz — COMMENTED** [review]
Excellent work on this @caroldelwing ! I have left a couple of comments which I hope you will find helpful ⭐

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 15]
```diff
@@ -0,0 +1,641 @@
...
+
+**Settings**
+
+The **Settings** section offers advanced options for OS patching, scheduled scans, scheduled backups, and cluster role
+binding. Utilize the default values, and click on the **Validate** button.
+
+**Review**
+
+The **Review** section allows you to review the cluster configuration before deploying the cluster. If everything is
+correct, click on **Finish Configuration** to start the cluster deployment, which may take up to 20 minutes.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:404]:
```suggestion
The **Review** section allows you to review the cluster configuration before deploying the cluster. Click on **Finish Configuration** to start the cluster deployment, which may take up to 20 minutes.
```
Let's assume it's correct, that way we can shorten the sentence.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:1]: We have two validate sections. Let's just do the validation at the end, with the deployed app.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 16]
```diff
@@ -0,0 +1,641 @@
...
+Next, copy your Palette API key and export it as an environment variable with the command below. Replace
+`YourAPIKeyHere` with your API key.
+
+```bash
+export SPECTROCLOUD_APIKEY=YourAPIKeyHere
+```
+
+### Create a Cluster Profile and Deploy a VMware Cluster
+
+Before proceeding, we suggest that you explore all the Terraform files present in the directory. Below is a high-level
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:443]:
```suggestion
Before proceeding, we recommend that you explore all the Terraform files present in the directory. Below is a high-level
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 17]
```diff
@@ -0,0 +1,641 @@
...
+  | `search_domain`      | Provide the name of the network search domain of your vSphere environment. For example, `spectrocloud.dev`. | Yes      |
+  | `vsphere_cluster`    | Provide the vSphere compute cluster to use for the VMware cluster deployment.                               | Yes      |
+  | `datastore_name`     | Provide the vSphere datastore name to use for the VMware cluster deployment.                                | Yes      |
+  | `network_name`       | Provide the vSphere network name to use for the VMware cluster deployment.                                  | Yes      |
+  | `resource_pool_name` | Provide the vSphere resource pool to target when deploying the VMware cluster.                              | Yes      |
+  | `ssh_key`            | Provide the path to your public SSH key. If not provided, a new key pair will be created.                   | No       |
+  | `ssh_key_private`    | Provide the path to your private SSH key. If not provided, a new key pair will be created.                  | No       |
+
+When you are ready making the required changes, initialize Terraform with the command presented below.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:498]: We need to make it clearer that the user needs to update the tfvars. We could add that as a paragraph before the current sentence or however you want to make that clear. But keeping that guidance in the bullet for tfvars is not ideal as some will skip reading those bullets.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 18]
```diff
@@ -0,0 +1,641 @@
...
+The `init` command will download the plugins and providers defined in the **provider.tf** file.
+
+Next, issue the `terraform plan` command to preview the resources that Terraform will create.
+
+```bash
+terraform plan
+```
+
+The output displays the resources that Terraform will create in an actual implementation, including the cluster profile,
+VMware vSphere cluster, and a new SSH key pair if the keys were not provided.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:513]:
```suggestion
The output displays the resources that Terraform will create in an actual implementation, including the cluster profile,
VMware vSphere cluster, and a new SSH key pair unless an existing key pair was specified.
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 13 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:516]: It's not something we have formalized or standardized on but it's out there.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 19]
```diff
@@ -0,0 +1,641 @@
...
+
+It can take up to 20 minutes to complete the cluster deployment process. Once completed, the following output will be
+displayed.
+
+```text hideClipboard
+# Output condensed for readability
+Apply complete! Resources: 5 added, 0 to changed, 0 to destroyed.
+```
+
+Now, log in to Palette, navigate to the left **Main Menu** and click on **Clusters**. While the deployment is in
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:534]:
```suggestion
Log in to Palette, navigate to the left **Main Menu**, and click on **Clusters**. While the deployment is in
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 20]
```diff
@@ -0,0 +1,641 @@
...
+progress, Palette displays the cluster status as **Provisioning**.
+
+![Palette's Clusters page.](/clusters_pcg_deploy-app-pcg_cluster-provisioning.png)
+
+You can select the `pcg-tutorial-cluster` to explore deployment status and event logs.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:539]:
```suggestion
You can select the **pcg-tutorial-cluster** to explore deployment status and event logs.
```
Switching to bold as it's a UI element.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 21]
```diff
@@ -0,0 +1,641 @@
...
+
+</TabItem>
+
+</Tabs>
+
+## Validate
+
+Once the cluster creation process has finished, log in to Palette, navigate to the left **Main Menu**, and click on
+**Clusters**. Select the `pcg-tutorial-cluster` to display the cluster's **Overview** page. Once the cluster status
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:548]:
```suggestion
**Clusters**. Select the **pcg-tutorial-cluster** to display the cluster's **Overview** page. Once the cluster status
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 22]
```diff
@@ -0,0 +1,641 @@
...
+sidebar_position: 50
+tags: ["pcg"]
+---
+
+Palette Private Cloud Gateway (PCG) is a crucial infrastructure support component that acts as a bridge between your
+private cloud environment or data center and Palette.
+
+A PCG is required in environments lacking direct network access to Palette. For example, many infrastructure
+environments reside within private networks that restrict external connections, preventing Palette from reaching them.
+Upon installation, the PCG acts as an endpoint, enabling Palette to securely deploy and delete Kubernetes clusters
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:19]: To Adelina's point, I think what we are missing the statement that explains that the PCG initaties a connection from inside the private network to Palette. The PCG polls Palette for instructions and when it's instructed to add or remove a cluster it does the work. Once the cluster is up, the PCG is no longer used as the cluster communicates directly with Palette (again initiating the connection to Palette).

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 23]
```diff
@@ -0,0 +1,641 @@
...
+
+## Wrap-Up
+
+In this tutorial, you learned how to deploy a VMware PCG using Palette CLI and utilize it to deploy a Palette cluster
+with a sample Kubernetes application to your VMware vSphere environment. Next, you accessed the Hello Universe
+application landing page through the exposed service URL.
+
+A PCG is a powerful component that enables Palette to communicate with private clouds or data center environments that
+impose restrictions on external connections. With a PCG, you can deploy Palette clusters in environments such as VMware
+vSphere, while also providing lifecycle support and monitoring for your clusters.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:634]: We should reinforce at the end that the PCG supports using Palette in restricted networks that don't allow inbound connectives from the outside. The PCG acts as the endpoint due to it's design as the source and polls Palette for instructions.  This is how we uses can use Palette without having to worry about making scary firewall rules or exposes public endpoints.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:19]: Got it! I rewrote the paragraph and provided more information.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 24]
```diff
@@ -0,0 +1,641 @@
...
+Palette Private Cloud Gateway (PCG) is a crucial infrastructure support component that acts as a bridge between your
+private cloud environment or data center and Palette.
+
+A PCG is required in environments lacking direct network access to Palette. For example, many infrastructure
+environments reside within private networks that restrict external connections, preventing Palette from reaching them.
+Upon installation, the PCG acts as an endpoint, enabling Palette to securely deploy and delete Kubernetes clusters
+within your private cloud environment or data center. Refer to the [PCG Architecture](./architecture.md) section for
+more information.
+
+Typically, a PCG is used with Palette SaaS and self-hosted Palette instances that do not have direct access to the
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:23]: I removed the System PCG paragraph after Karl's suggestion. :)

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 25]
```diff
@@ -0,0 +1,641 @@
...
+target environment. However, a System Private Gateway is recommended when direct network connectivity is available.
+Visit the [System Private Gateway](./architecture.md#system-private-gateway) section to learn more.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:25]: I agree. Done!

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 26]
```diff
@@ -0,0 +1,641 @@
...
+Once issued, you will be prompted for several parameters to complete the authentication. The table below outlines the
+required parameters along with the values that will be utilized in this tutorial. If a parameter is specific to your
+environment and Palette account, such as your Palette API key, ensure to input the value according to your environment.
+Access the [Deploy a PCG to VMware vSphere](../pcg/deploy-pcg/vmware.md) guide for detailed information about each
+option.
+
+| **Parameter**                  | **Value**                                                                                                        | **Environment-Specific** |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| **Spectro Cloud Console**      | `https://console.spectrocloud.com`. If using a self-hosted instance of Palette, enter the URL for that instance. | No                       |
+| **Allow Insecure Connection**  | `Y`                                                                                                              | No                       |
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:93]: Added a lil note :)

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 3 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:231]: Yes!

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 4 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:237]: :white_check_mark:

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 6 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:246]: :white_check_mark:

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 27]
```diff
@@ -0,0 +1,641 @@
...
+Settings Menu**. Click on the PCG cluster you just deployed to access its **Overview** page and check the deployment
+progress under the **Events** tab.
+
+![PCG Events page.](/clusters_pcg_deploy-app-pcg_pcg-events.png)
+
+Additionally, you can observe the PCG deployment progress from your terminal. Upon completion, the local kind cluster is
+automatically deleted from your machine.
+
+![Palette CLI PCG deployment](/clusters_pcg_deploy-app-pcg_pcg-cli.png)
+
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:256]: :white_check_mark:

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:1]: I removed the first **Validate** section, but I kept the PCG status verification as this is required for the cluster deployment step.

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 28]
```diff
@@ -0,0 +1,641 @@
...
+| **Spectro Cloud Console**      | `https://console.spectrocloud.com`. If using a self-hosted instance of Palette, enter the URL for that instance. | No                       |
+| **Allow Insecure Connection**  | `Y`                                                                                                              | No                       |
+| **Spectro Cloud API Key**      | Enter your Palette API Key.                                                                                      | Yes                      |
+| **Spectro Cloud Organization** | Select your Palette Organization name.                                                                           | Yes                      |
+| **Spectro Cloud Project**      | `None (TenantAdmin)`                                                                                             | No                       |
+| **Acknowledge**                | `Y`                                                                                                              | No                       |
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:97]: :white_check_mark: :)

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 17 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:495]: Excellent point!

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 23 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:631]: I rewrote the paragraph :)

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 2 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:112]: Using the prod-automation account still displays `cardenas88karl-gmail` when selecting the org.
CleanShot 2024-03-13 at 10 17 39
I increased the playback speed and used the stage account instead. What do you think?

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 7 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:243]: The playback speed was increased and the recording now is under 1 min :)

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 29]
```diff
@@ -625,13 +625,14 @@ your VMware vSphere environment.
 
 ## Wrap-Up
 
-In this tutorial, you learned how to deploy a VMware PCG using Palette CLI and utilize it to deploy a Palette cluster
-with a sample Kubernetes application to your VMware vSphere environment. Next, you accessed the Hello Universe
-application landing page through the exposed service URL.
+In this tutorial, you learned how to deploy a VMware PCG using Palette CLI. Then, you used the PCG to support the
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:628]:
```suggestion
In this tutorial, you learned how to deploy a VMware PCG using the Palette CLI. Then, you used the PCG to support the
```

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 30]
```diff
@@ -625,13 +625,14 @@ your VMware vSphere environment.
...
+deployment of a Palette cluster with a sample Kubernetes application to your VMware vSphere environment. Next, you
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:629]:
```suggestion
deployment of a Kubernetes cluster with a sample application to your VMware vSphere environment. Next, you
```

Removing the term "Palette cluster" as we are deploying a Kubernetes cluster. And the application is just a regular application :)

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md [hunk 31]
```diff
@@ -86,27 +89,25 @@ palette login
...
+After accepting the login banner message, you will receive the following output confirming you have successfully
+authenticated with Palette.
 
 ```text hideClipboard
 Welcome to Spectro Cloud Palette
 ```
 
-The video below demonstrates Palette's authentication process. Ensure you utilize values appropriate for your
-environment.
+The video below demonstrates Palette's authentication process. Ensure you utilize values specific to your environment.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/pcg/deploy-app-pcg.md:110]:
```suggestion
The video below demonstrates Palette's authentication process. Ensure you utilize values specific to your environment, such as the correct Palette URL. Contact your Palette administrator for the correct URL if you use a self-hosted Palette or VerteX instance. 
```
Thoughts on making the URL part a bit clearer?

**Review by karl-cardenas-coding — APPROVED** [review]
Awesome tutorial 👏🏻

File: docs/docs-content/clusters/pcg/deploy-app-pcg.md (hunk 31 above)
Comment (caroldelwing) [reply docs/docs-content/clusters/pcg/deploy-app-pcg.md:110]: I liked it :)

**Pull request merged**